libyul/optimiser/CallGraphGenerator.cpp

```diff
@@ -29,26 +29,38 @@ using namespace std;
 using namespace dev;
 using namespace yul;
 
+map<YulString, set<YulString>> CallGraphGenerator::callGraph(Block const& _ast)
+{
+	CallGraphGenerator gen;
+	gen(_ast);
+	return gen.m_callGraph;
+}
 
 void CallGraphGenerator::operator()(FunctionalInstruction const& _functionalInstruction)
 {
-	m_callGraph.insert(m_currentFunction, YulString{
-		dev::eth::instructionInfo(_functionalInstruction.instruction).name
-	});
+	string name = dev::eth::instructionInfo(_functionalInstruction.instruction).name;
+	std::transform(name.begin(), name.end(), name.begin(), [](unsigned char _c) { return tolower(_c); });
+	m_callGraph[m_currentFunction].insert(YulString{name});
 	ASTWalker::operator()(_functionalInstruction);
 }
 
 void CallGraphGenerator::operator()(FunctionCall const& _functionCall)
 {
-	m_callGraph.insert(m_currentFunction, _functionCall.functionName.name);
+	m_callGraph[m_currentFunction].insert(_functionCall.functionName.name);
 	ASTWalker::operator()(_functionCall);
 }
 
 void CallGraphGenerator::operator()(FunctionDefinition const& _functionDefinition)
 {
 	YulString previousFunction = m_currentFunction;
 	m_currentFunction = _functionDefinition.name;
+	m_callGraph[m_currentFunction];
 	ASTWalker::operator()(_functionDefinition);
 	m_currentFunction = previousFunction;
 }
 
+CallGraphGenerator::CallGraphGenerator()
+{
+	m_callGraph[YulString{}];
```

Comment:
```suggestion
	m_callGraph[YulString{}] = {};
```
It's the same, but maybe easier to understand?
Or otherwise member initialization in the class like
``
std::map<YulString, std::set<YulString>> m_callGraph = {{YulString{}, {}}};
``
and no constructor (or a private, but defaulted one)?